### PR TITLE
Develop the main application template.

### PR DESCRIPTION
**Pull request by evvis**
Develop the main application template. Using the library matrializecss, adding to the project.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,8 +7,39 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0-rc.2/css/materialize.min.css" rel="stylesheet">
 </head>
 <body>
-  <app-root></app-root>
+  <div>
```
Comment (maximprokopchuk) [inline src/index.html]: Move header into separate component that used in root component

File: src/index.html [hunk 2]
```diff
@@ -7,8 +7,39 @@
...
+              <i class="material-icons">close</i>
+            </div>
+          </form>
+        </div>
+      </nav>
+    </header>
+    <app-root></app-root>
+    <footer>
+      <div class="footer">
+        <ul class="pagination">
```
Comment (maximprokopchuk) [inline src/index.html]: No need for pagination now

File: src/index.html [hunk 3]
```diff
@@ -7,8 +7,39 @@
...
 <body>
-  <app-root></app-root>
+  <div>
+    <header class="header">
+      <nav>
+        <div class="nav-wrapper">
+          <form>
+            <div class="input-field">
+              <a href="#!" class="brand-logo">TheMovieDB</a>
+              <input id="search" type="search" required>
```
Comment (maximprokopchuk) [inline src/index.html]: Not sue about search now. Let's implement it later

File: src/app/app.component.html [hunk 4]
```diff
@@ -1,20 +1,41 @@
...
+          <a class="btn-floating halfway-fab waves-effect waves-light red"><i class="material-icons">add</i></a>
+        </div>
+        <div class="card-content">
+          <p>I am a very simple card. I am good at containing small bits of information. I am convenient because I require little markup to use effectively.</p>
+        </div>
+      </div>
+    </div>
+    <div class="col s3">
+      <div class="card">
+        <div class="card-image">
```
Comment (maximprokopchuk) [inline src/app/app.component.html]: This cards is extra markup now. Remove it. But looks nice

**Review by maximprokopchuk — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (evvis) [reply src/index.html]: Changed

File: src/index.html (hunk 3 above)
Comment (evvis) [reply src/index.html]: Search deleted

File: src/app/app.component.html (hunk 4 above)
Comment (evvis) [reply src/app/app.component.html]: Made for clarity. Has removed.

**Comment by evvis** [comment]
All changes are made according to the recommendations.

File: src/app/app.component.html [hunk 5]
```diff
@@ -1,20 +1,17 @@
-<!--The content below is only a placeholder and can be replaced.-->
-<div style="text-align:center">
-  <h1>
-    Welcome to {{ title }}!
-  </h1>
-  <img width="300" alt="Angular Logo" src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCAyNTAgMjUwIj4KICAgIDxwYXRoIGZpbGw9IiNERDAwMzEiIGQ9Ik0xMjUgMzBMMzEuOSA2My4ybDE0LjIgMTIzLjFMMTI1IDIzMGw3OC45LTQzLjcgMTQuMi0xMjMuMXoiIC8+CiAgICA8cGF0aCBmaWxsPSIjQzMwMDJGIiBkPSJNMTI1IDMwdjIyLjItLjFWMjMwbDc4LjktNDMuNyAxNC4yLTEyMy4xTDEyNSAzMHoiIC8+CiAgICA8cGF0aCAgZmlsbD0iI0ZGRkZGRiIgZD0iTTEyNSA1Mi4xTDY2LjggMTgyLjZoMjEuN2wxMS43LTI5LjJoNDkuNGwxMS43IDI5LjJIMTgzTDEyNSA1Mi4xem0xNyA4My4zaC0zNGwxNy00MC45IDE3IDQwLjl6IiAvPgogIDwvc3ZnPg==">
+<header class="header all_fields">
```
Comment (maximprokopchuk) [inline src/app/app.component.html]: Make separate commit for header and footer and use this new component there

File: src/app/app.component.html [hunk 6]
```diff
@@ -1,20 +1,17 @@
...
+  <div class="right-align">
```
Comment (proroker) [inline src/app/app.component.html]: Use consistent paddings in the code

File: src/index.html [hunk 7]
```diff
@@ -7,8 +7,12 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0-rc.2/css/materialize.min.css" rel="stylesheet">
```
Comment (maximprokopchuk) [inline src/index.html]: No need for this. Use installed package instead

**Review by maximprokopchuk — CHANGES_REQUESTED** [review]

**Comment by evvis** [comment]
making changes, splitting into components

File: src/index.html [hunk 8]
```diff
@@ -7,8 +7,11 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
 </head>
 <body>
-  <app-root></app-root>
+  <div>
```
Comment (maximprokopchuk) [inline src/index.html]: why 'div' wrapper for app-root?

Comment (evvis) [reply src/index.html]: he's really superfluous, fixed

**Comment by evvis** [comment]
fixed.

**Review by maximprokopchuk — APPROVED** [review]
